src/rules/cognitive-complexity.ts

```diff
@@ -113,6 +113,9 @@ const rule: TSESLint.RuleModule<string, (number | 'metric' | 'sonar-runtime')[]>
     /** Stack of enclosing functions */
     const enclosingFunctions: TSESTree.FunctionLike[] = [];
 
+    /** Indicator if inside a JSX expression */
+    let jsx = false;
```

Comment:
`isJsx`?

Reply:
Renamed to jsxLevel as this should be a number.